examples/kubernetes/deployment+service.privileged.yaml

```diff
@@ -28,6 +28,22 @@ spec:
         - /certs/cert.pem
         - --tlskey
         - /certs/key.pem
+        readinessProbe:
```

Comment:
maybe better to add a comment to clarify that this works for master but does not work v0.6.3

Reply:
Comment added